busylight/api/busylight_api.py

```diff
@@ -84,24 +81,25 @@ def __init__(self):
             self.username = None
             self.password = None
 
-        logger.info(
-            "Set up CORS Access-Control-Allow-Origin header, if environment variable is set.")
-        try:
-            self.origins = json_loads(
-                environ["BUSYLIGHT_API_CORS_ORIGINS_LIST"])
-            logger.info(
-                "CORS Access-Control-Allow-Origin list: {}".format(self.origins))
-            logger.info("Found CORS allowed origins in environment.")
-        except KeyError:
-            logger.info("Did NOT find CORS allowed origins in environment.")
+        # Get and save the CORS Access-Control-Allow-Origin header
+        logger.info("Set up CORS Access-Control-Allow-Origin header, if environment variable is set.")
```

Comment:
It would be kind to users to mention the name of environment variable that is expected here. This keeps them from digging thru documentation or reading the source.